README.md

```diff
@@ -11,27 +11,47 @@ will work for everything given the potential abuses of the file format
 ### How do I get set up? ###
 
 * Python interpreter (almost any version will do)
-* PIP (only if you need to run the tests)
+* Either pip, [pipx](https://pipxproject.github.io/pipx/), or the repository to your local computer (see [instructions](#instructions))
 * Some suitable BASIC CAS files
-* Clone the repository to your local computer
 
-## Instructions ##
+## Instructions
 
-To use the script just run ```python cas2bas.py``` followed by the name of
-your CAS file
+There are a few options for getting the script working.
+
+### Using pipx
+
+If you use [pipx](https://pipxproject.github.io/pipx/) you can run
+
+```pipx install git+https://github.com/jimbro1000/cas2bas.git@development```
```

Comment:
I need to check this further but I'm seeing packaging errors following the instructions (obviously not verbatim as the package doesn't exist on this path yet). This may not be an issue with the package as I also have issues using pipx under windows